app/routers/features.py

```diff
@@ -0,0 +1,152 @@
+from fastapi import APIRouter, Request, Depends
+from typing import List
+
+from app.dependencies import get_db, SessionLocal
+from app.database.models import UserFeature, Feature
+from app.internal.features import (
+    create_association,
+    is_association_exists_in_db,
+    is_feature_exists_in_disabled,
+    is_feature_exists_in_enabled,
+    get_user_disabled_features,
+    get_user_enabled_features
+)
+
+router = APIRouter(
+    prefix="/features",
+    tags=["event"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+@router.get('/')
+async def index(
+    request: Request, session: SessionLocal = Depends(get_db)
+) -> List:
+    features = session.query(Feature).all()
+    return features
+
+
+@router.post('/add')
+async def add_feature_to_user(
+    request: Request, session: SessionLocal = Depends(get_db)
+) -> UserFeature:
+    form = await request.form()
+
+    user_id = form['user_id']  # OPTION - get active user id instead.
```

Comment:
Currently it is possible to add feature for other users. Please fix it.

Reply:
Done.